addons/l10n_sa_edi/views/account_tax_views.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_tax_form" model="ir.ui.view">
+        <field name="name">account.tax.form.zatca</field>
+        <field name="model">account.tax</field>
+        <field name="inherit_id" ref="account.view_tax_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='country_id']" position="after">
+                <field name="l10n_sa_is_retention"/>
```

Comment:
A tax has a company_id normally so we can make it invisible if it is not an SA company.

Reply:
@jco-odoo done